### PR TITLE
Added Go Template

### PR DESCRIPTION
**Pull request by LordMartron94**
Also improved the general readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
 
-# Contributing
+## Contributing
 
 Want to add a template for a new language? Feel free to open a pull request!
+
+### Contributions:
+- C#, JavaScript, Python: _Kamiel de Visser_
+- Kotlin: _NoahMelle_
+- Dosato: _RobotNik08_
```
Comment (Robotnik08) [inline README.md]: My name is spelled wrong

it should be: "Robotnik08"

Comment (LordMartron94) [reply README.md]: Fixed it.

**Comment by codevogel** [comment]
Can you remove the changes to `README.md` please? Otherwise, this looks great.

Data about the contributors can already be found [here](https://github.com/codevogel/adventofcode-templates/graphs/contributors). Adding that section to the `README` just adds extra work for future PRs.

Furthermore, a Table of Contents is a bit overkill for a README of this size.

**Comment by LordMartron94** [comment]
> Can you remove the changes to `README.md` please? Otherwise, this looks great.
> 
> Data about the contributors can already be found [here](https://github.com/codevogel/adventofcode-templates/graphs/contributors). Adding that section to the `README` just adds extra work for future PRs.
> 
> Furthermore, a Table of Contents is a bit overkill for a README of this size.

Done :)

**Comment by codevogel** [comment]
Thanks for contributing 🎉

**Pull request merged**